src/script/interpreter.cpp

```diff
@@ -493,6 +493,26 @@ static bool CheckSignatureEncodingSigHashChoice(const vector<unsigned char> &vch
     {
         return true;
     }
+
+    if (flags & SCRIPT_ENABLE_SCHNORR)
+    {
+        if (vchSig.size() == 64 + (check_sighash == true) ? 1 : 0) // 64 sig length plus 1 sighashtype
+        {
+            // In a generic-signature context, 64-byte signatures are interpreted
+            // as Schnorr signatures (always correctly encoded) when flag set.
+            if (check_sighash && ((flags & SCRIPT_VERIFY_STRICTENC) != 0))
```

Comment:
and this? (just curious why all the flag checks sometimes compare against something or not)
`if (check_sighash && (flags & SCRIPT_VERIFY_STRICTENC))`

Reply:
Its a matter of individual style and experience with other languages I think. Some people prefer to be very explicit (me), others rely on the fact that x & y may evaluate to nonzero and any nonzero value is cast to boolean true, while zero is cast to false.

I don't think that BU has a coding standard on this.